htdocs/admin/modulehelp.php

```diff
@@ -451,10 +451,10 @@
     }
     require_once DOL_DOCUMENT_ROOT.'/core/class/interfaces.class.php';
     $interfaces = new Interfaces($db);
-    $triggers = $interfaces->getTriggersList(array((($objMod->isCoreOrExternalModule() == 'external')?'/'.$moduledir:'').'/core/triggers'));
+    $triggers = $interfaces->getTriggersList(array((($objMod->isCoreOrExternalModule() == 'external')?'/'.$objMod->code:'').'/core/triggers'));
```

Comment:
??
This property ->code seems not defined for objects that herits DolibarrModules.